qualtran/bloqs/mcmt/multi_target_cnot.py

```diff
@@ -64,6 +65,13 @@ def _circuit_diagram_info_(self, _) -> cirq.CircuitDiagramInfo:
             raise ValueError(f'Symbolic bitsize {self.bitsize} not supported')
         return cirq.CircuitDiagramInfo(wire_symbols=["@"] + ["X"] * self.bitsize)
 
+    def on_classical_vals(
+        self, control: 'ClassicalValT', targets: 'ClassicalValT'
+    ) -> Dict[str, 'ClassicalValT']:
+        if control:
+            targets = (2**self.bitsize - 1) ^ targets
```

Comment:
can you add a test for this classical action

Reply:
done